packages/fetchai/skills/simple_data_request/handlers.py

```diff
@@ -63,6 +63,7 @@ def handle(self, message: Message) -> None:
         # recover dialogue
         http_dialogues = cast(HttpDialogues, self.context.http_dialogues)
         http_dialogue = cast(HttpDialogue, http_dialogues.update(http_msg))
+
```

Comment:
please revert to avoid unnecessary bumping

Reply:
done